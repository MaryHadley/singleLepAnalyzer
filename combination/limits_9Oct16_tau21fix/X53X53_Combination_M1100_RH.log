000 (1295359.000.000) 10/09 22:41:04 Job submitted from host: <192.168.63.249:41533>
...
001 (1295359.000.000) 10/09 22:41:05 Job executing on host: <192.168.63.239:46789>
...
006 (1295359.000.000) 10/09 22:41:14 Image size of job updated: 11336
	2  -  MemoryUsage of job (MB)
	1236  -  ResidentSetSize of job (KB)
...
006 (1295359.000.000) 10/09 22:46:15 Image size of job updated: 283676
	142  -  MemoryUsage of job (MB)
	144556  -  ResidentSetSize of job (KB)
...
006 (1295359.000.000) 10/10 00:01:23 Image size of job updated: 283676
	142  -  MemoryUsage of job (MB)
	144640  -  ResidentSetSize of job (KB)
...
006 (1295359.000.000) 10/10 19:48:26 Image size of job updated: 297936
	158  -  MemoryUsage of job (MB)
	161032  -  ResidentSetSize of job (KB)
...
005 (1295359.000.000) 10/10 19:53:27 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 21:10:04, Sys 0 00:00:05  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 21:10:04, Sys 0 00:00:05  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	1596  -  Run Bytes Sent By Job
	307  -  Run Bytes Received By Job
	1596  -  Total Bytes Sent By Job
	307  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :    31691        1   3824810
	   Memory (MB)          :      158     3072      3072
...
