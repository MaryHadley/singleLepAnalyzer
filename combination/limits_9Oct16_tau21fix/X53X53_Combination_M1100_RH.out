/var/lib/condor/execute/dir_15257
Total number of likelihood evaluations: 1100000101
Total number of likelihood evaluations: 11000101
[INFO]  executing script /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M1100_RH.py
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M1100_RH/quant-toys:0-sig-4373177346.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M1100_RH/quant-data-sig-148ec13493.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  workdir is /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M1100_RH
