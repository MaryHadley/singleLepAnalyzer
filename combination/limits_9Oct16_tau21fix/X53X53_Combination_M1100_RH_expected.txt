# x; y; band 0 low; band 0 high; band 1 low; band 1 high
         0   0.071548   0.035778    0.14957  0.049398     0.1052
